app/models/provisioning_template.rb

```diff
@@ -190,4 +190,14 @@ def self.pxe_default_combos
     end
     combos
   end
+
+  def self.local_boot_template(host)
+    return unless host.operatingsystem.present?
```

Comment:
Sorry, I'm confused as to why can you not leave this to `host.provisioning_template`? It's the way the rest of the code retrieves it, you could add something like:

```ruby
return local_boot_template if opts[:kind] == 'local_boot'
``` 

over here https://github.com/theforeman/foreman/blob/develop/app/models/host/managed.rb#L327